index.html

```diff
@@ -0,0 +1,82 @@
+    <link rel="stylesheet" href="css/styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f0e9415ee2.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>William Raga</title>
+  </head>
+  <body>
+    <header>
+      <h4 class="white text-normal align-center">William Raga</h4>
+      <nav>
+        <i class="fa-solid fa-bars fa-2x white"></i>
+      </nav>
+    </header>
+
+    <main>
+      <div class="white-background"></div>
+      <div class="my-title">
+        <h1 class="head-color align-center">William Raga</h1>
+        <p class="text-normal align-center text-color">Software Developer</p>
+      </div>
+      <div>
+        <p class="text-400 align-center description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <hr class="horizontal" />
+      <div class="qualifications">
+        <div class="white-background"></div>
+        <div>
+          <h4 class="head-color text-normal align-left">Languages</h4>
+          <ul>
+            <li class="text-400 align-left li-color">JavaScript</li>
+            <li class="text-400 align-left li-color">Ruby</li>
+            <li class="text-400 align-left li-color">Html</li>
+            <li class="text-400 align-left li-color">Css</li>
+          </ul>
+        </div>
+        <div>
+          <h4 class="head-color text-normal">Frameworks</h4>
+          <ul>
+            <li class="text-400 align-left li-color">Bootstrap</li>
+            <li class="text-400 align-left li-color">Ruby on rails</li>
+            <li class="text-400 align-left li-color">RSPec</li>
+            <li class="text-400 align-left li-color">CapyBara</li>
+            <li class="text-400 align-left li-color">Selenium</li>
+          </ul>
+        </div>
+        <div>
+          <h4 class="head-color text-normal align-left">Skills</h4>
+          <ul>
+            <li class="text-400 align-left li-color">Codekit</li>
+            <li class="text-400 align-left li-color">GitHub</li>
+            <li class="text-400 align-left li-color">Codepen</li>
+            <li class="text-400 align-left li-color">GitLab</li>
+            <li class="text-400 align-left li-color">Terminal</li>
+          </ul>
+        </div>
+      </div>
+      <hr class="horizontal" />
+      <div class="socials">
+        <a href="https://github.com/itsmraga-hub"
+          ><i class="fa fa-github white" aria-hidden="true"></i
+        ></a>
+        <a href="https://github.com/itsmraga-hub"
+          ><i class="fa fa-linkedin white" aria-hidden="true"></i
+        ></a>
+        <a href="https://twitter.com/RagaMacharia"
+          ><i class="fa fa-twitter white" aria-hidden="true"></i
+        ></a>
+      </div>
+    </main>
+  </body>
+</html>
```

Comment:
- [ ] It's not easy to review your project results because your CSS is not running successfully, just make sure that you've followed the provided requirements for you to go to the next milestone

Reply:
That was my bad @ishimwezachee . I'm still grasping mobile-first, min-width, and max-width concepts.